### PR TITLE
Review of pc files integration

### LINKED ISSUE #818: Minor documentation error in AutoTools example
There is an AutoTools example at the bottom of this page:  https://docs.conan.io/en/latest/integrations/pkg_config_pc_files.html

The `import` statement is messed up.  You are importing `Meson` but it should be `AutoToolsBuildEnvironment` I think.

### PR DESCRIPTION
**Pull request by danimtb**
closes #818

### CODE REVIEW

File: integrations/pkg_config_pc_files.rst [hunk 1]
```diff
@@ -47,16 +42,17 @@ Example of a ``.pc`` file with an absolute path:
     Libs: -L${libdir} -L${sharedlibdir} -lz
     Cflags: -I${includedir}
 
+To solve this problem there are different approaches that can be followed.
```
Comment (jgsogo) [inline integrations/pkg_config_pc_files.rst]: Those are just alternative solutions, or do Conan prefer one over another? I would state that he preferred one is...

Comment (danimtb) [reply integrations/pkg_config_pc_files.rst]: Up to the user I would say. This describes just ways of intergating pc files with Conan

**Comment by jgsogo** [comment]
@danimtb, can you explain to me what is the objective of each branch here in docs? I don't know why this pull request should be to `master` branch instead of `develop`... because it is more like a bugfix? are the docs deployed with every commit to `master`?

Thanks!

**Comment by danimtb** [comment]
@jgsogo Only changes that are specific to the new Conan release go to develop, improvements to current docs, bugs or missing documentation go to master.

Yes, every new commit into master will update and deploy the latest version of the docs

File: integrations/pkg_config_pc_files.rst [hunk 2]
```diff
@@ -3,33 +3,28 @@
 pkg-config and pc files
 =======================
 
-Intro
------
-
 If you are creating a Conan package for a library (A) and the build system uses ``.pc`` files to locate
```
Comment (jgsogo) [inline integrations/pkg_config_pc_files.rst]: Use `*.pc*`or ` ``.pc`` ` consistently across the document

File: integrations/pkg_config_pc_files.rst [hunk 3]
```diff
@@ -3,33 +3,28 @@
...
 
     /custom/dir/.data/storage/zlib/1.2.11/conan/testing
 
-So the ``.pc``  files containing absolute paths won't work to locate the dependencies.
-
-
-Example of a ``.pc`` file with an absolute path:
+You can see that *.pc* files containing absolute paths won't work to locate the dependencies.
 
+Example of a **.pc** file with an absolute path:
```
Comment (jgsogo) [inline integrations/pkg_config_pc_files.rst]: Idem: use consistently `.pc`.

File: integrations/pkg_config_pc_files.rst (hunk 2 above)
Comment (jgsogo) [reply integrations/pkg_config_pc_files.rst]: Personally I prefer the source approach than the italics one.

Comment (jgsogo) [reply integrations/pkg_config_pc_files.rst]: Or maybe the `:command:``pkg-config`` ` which produces a different output...

**Comment by danimtb** [comment]
Sticking this PR to the style guidelines: https://github.com/conan-io/docs#style-guidelines

File: integrations/pkg_config_pc_files.rst [hunk 4]
```diff
@@ -137,24 +128,22 @@ Library A recipe (importing and consuming ``pc`` file):
                 # Or directly declare the variables:
                 self.run('g++ main.cpp $(pkg-config %s libB --libs --cflags) -o main' % args)
 
+Approach 3: Use :command:`--define-prefix`
+-----------------------------------------
```
Comment (jgsogo) [inline integrations/pkg_config_pc_files.rst]: Here we need one more dash `-`.

**Pull request merged**